src/Pages/Landing/Landing.test.js

```diff
@@ -3,6 +3,6 @@ import Landing from './Landing';
 
 test('renders learn react link', () => {
   render(<Landing/>);
-  const linkElement = screen.getByText(/Landing/i);
+  const linkElement = screen.getByTestId(/main-content/i);
```

Comment:
Here could be checked that the landing page has required components like Header, Login form and Footer

Reply:
But that is not really topic of this issue. I'll open separete issue for that